qutebrowser/config/configdata.yml

```diff
@@ -1768,6 +1768,11 @@ tabs.pinned.shrink:
   type: Bool
   desc: Shrink pinned tabs down to their contents.
 
+tabs.pinned.frozen:
+    type: Bool
+    default: True
+    desc: If true, then pinned tabs cannot have their URL changed.
```

Comment:
Could you re-write this to imperative mood for consistency with other options (see https://github.com/qutebrowser/qutebrowser/commit/494aceec451e307b0227b0fdd0f86d419dbc11d1)?
Maybe `Force pinned tabs to stay at a fixed URL.`?

Reply:
Alright. Done.